### ISSUE TITLE
Invalid configuration of .ddev/config.yaml

### ISSUE THREAD

**Issue by nox1134** [mautic/mautic#13565]
### Mautic Version

5.0.x series

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.2

### What browsers are you seeing the problem on?

Not relevant

### What happened?

While setting up the Mautic local environment, I ran the command `ddev start` and got the following error:
```
ddev start
Failed to start project(s): /Users/priyanshi/opensource/gsoc/mautic/mautic/.ddev/config.yaml exists but cannot be read. It may be invalid due to a syntax error: unable to load config file /Users/priyanshi/opensource/gsoc/mautic/mautic/.ddev/config.yaml: invalid configuration in /Users/priyanshi/opensource/gsoc/mautic/mautic/.ddev/config.yaml: yaml: line 263: could not find expected ':'
```
While checking the commit history, I found that the .ddev/config.yaml file was recently changed (https://github.com/mautic/mautic/pull/12818).

I verified the config.yaml file using the [online YAML validator](https://yamlchecker.com/) and received the following error: 
<img width="997" alt="image" src="https://github.com/mautic/mautic/assets/95358802/2980f462-60f7-4977-a74a-1d33035c33f1">
It appears there is an issue with the YAML syntax.

### How can we reproduce this issue?

Step 1: Clone the github repo.
Step 2: Start the dev environment using ddev start 

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Oof thanks for reporting this @nox1134 - this update was generated with the DDEV feature documented here: https://twitter.com/randyfay/status/1713897478364279272?t=AdJfKrJViDjPNYH6q1u2vw&s=09 and it worked previously but was some time ago. I'll revert the PR initially so we can fix.

**Comment by RCheesley** [comment]
REsolved with #13567 - we'll get back to updating the config file at another time.
